db/migrate/20180419130340_add_parent_id_to_search_queries.rb

```diff
@@ -0,0 +1,8 @@
+class AddParentIdToSearchQueries < ActiveRecord::Migration[5.0]
+  def change
+    add_column :search_queries, :parent_id, :integer
+    add_column :search_queries, :query_type, :search_query_type, null: false, default: :search
+    add_column :search_queries, :filter_type, :string
```

Comment:
I think the enums `query_type` and `filter_type` need to be defined, I guess they got stripped out in the latest refactoring but still exist in your local DB.

Reply:
query type yes,  Filter type isn't an enum.  Will fix it.